SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -1,32 +1,54 @@
+## Before you start
 
-> This workflow is very subject to change. We're aware that it's not optimized, and one of our priorities is to improve iteration times.
+Before porting your project: 
+* Install and set up the SpatialOS GDK: follow the [set up guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**. 
+* Make sure your Spatial CLI is up to date: from a terminal window, run the command `spatial update`.
+<!-- // TODO: Update the set up link when ready -->
 
-## Prerequisites
+### Terms used in this guide
+`<ProjectRoot>` - The root folder of your Unreal project.  
+`<GameRoot>` - The folder containing your game project's `.uproject` and source folder (for example, `<ProjectRoot>/ShooterGame/`).  
+`<YourProject>` - Name of your game project's `.uproject` (for example, `StarterProject.uproject`).
+
+## Start porting!
 
 <%(TOC)%>
 
-Follow the [setup guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
-> To speed up build times, install IncrediBuild, FastBuild, or another build distributor.
-
-## Setting up the project structure
-1. In a terminal window, navigate to the root directory of the SpatialOS Unreal GDK repository you cloned and run `Setup.bat`. This requires authorization with your SpatialOS account.
-1. Open the `StarterProject` repo and copy everything except the `\Game\` folder alongside your `<GameRoot>` folder. 
-Your folder structure should now resemble -
-```
-\<ProjectRoot>\ci\
-\<ProjectRoot>\<GameRoot>\
-\<ProjectRoot>\spatial\
-\<ProjectRoot>\ProjectPaths.bat
-\<ProjectRoot>\LaunchClient.bat 
-etc...
-```
-1. Open **`ProjectPaths.bat`** for editing and replace `Game` with `<GameRoot>`, and `StarterProject` with the name of your uproject (which we'll refer to as `<YourProject>`).
-
-## Adding the SpatialGDK module to your project
-1. In your project's `*.build.cs` file, add "SpatialGDK" to the `PublicDependencyModuleNames`.
+### 1. Set up the project structure
+1. Your game's project needs some extra files and folders to run with the GDK; you can copy these from the StarterProject repository that you cloned earlier.
+
+    To do this: either in a terminal window or your file manager, navigate to the root of the `StarterProject` repository and copy all of the files and folders below:  
+
+    ``` cpp
+    \StarterProject\spatial\
+    \StarterProject\LaunchClient.bat 
+    \StarterProject\LaunchServer.bat
+    \StarterProject\ProjectPaths.bat
+    ```
+    Your game's folder structure should now resemble:
+
+    ``` cpp
+    \<ProjectRoot>\<GameRoot>\
+    \<ProjectRoot>\spatial\
+    \<ProjectRoot>\LaunchClient.bat 
+    \<ProjectRoot>\LaunchServer.bat
+    \<ProjectRoot>\ProjectPaths.bat
+    etc...
+    ```
+
+1. Our helper scripts require configuration to work correctly. Set up your project paths:
+   
+   Open **`\<ProjectRoot>\ProjectPaths.bat`** for editing and:
```

Comment:
What sort of errors could users expect if they don't update this .bat file?

Reply:
Added some extra info